apps/motionscope/analyze.py

```diff
@@ -119,6 +124,12 @@ def func(val):
             self.curr_first_index, self.curr_last_index = val
             self.render()
 
+    def exportGS(self,filename):
```

Comment:
Maybe I am a style guy.... please put spaces between comma and next argument, list item....